Files/Services/Implementation/FileTagsSettingsService.cs

```diff
@@ -33,6 +33,11 @@ public IList<FileTag> FileTagList
 
         public FileTag GetTagById(string uid)
         {
+            if (FileTagList.Any(x => x.Uid == null))
```

Comment:
.IsNullOrEmpty() ?

Reply:
I need to check if Filelist contains any invalid items (items with a null ID).
I'm considering removing this as I'm not certain solves the related issue.